### ISSUE TITLE
`draft up` should tag and release a ":latest" tag

### ISSUE THREAD

**Issue by thedrow** [Azure/draft#541]
Draft makes it very hard to reuse the deployed image in other pods because the tag is neither provided during `draft up` nor tagged as latest.
I suggest to add the ability to tag the built image so that other pods may be able to use it.

I'm currently using pachyderm which has it's own way to deploy pods to Kubernetes so I can't use a helm chart to deploy them as of yet.

**Comment by bacongobbler** [comment]
Hey @thedrow, can you elaborate a little more on how we can make this possible? I'd love to hear more about your use case.

Is this the ability to re-use an image tag from another application? If it's from the same application, the image is available via Helm's values. 

https://github.com/Azure/draft/blob/master/docs/reference/dep-002.md#qa

> Keep in mind that there are certain values injected from Draftd into the chart which you'll likely want to use:

```yaml
image:
  repository: quay.io/bacongobbler/myapp  # the full name of the image
  tag: 08db751                            # the release of the image in the registry
basedomain: example.com                   # the base domain configured with the ingress controller
ondraft: true                             # metadata to demonstrate this was deployed using Draft
```

So in your chart, you can reference the built image via `{{ .Values.image.repository }}:{{ .Values.image.tag }}`.

Let us know if you were thinking of something else!

**Comment by thedrow** [comment]
I'm reusing the same image I built with a system that doesn't use the normal YAML files Kubernetes uses to create resources. Since Helm is not aware of the pipeline specification of Pachyderm I cannot use Draft to deploy those pipelines. At the very least, I need draft to tag the latest build as "latest" so I'll be able to reuse the same images in Pachyderm's pipelines.

**Comment by bacongobbler** [comment]
updated the title.

**Comment by radu-matei** [comment]
Note that if you do `draft logs` you should now see the tag of your image in the push step - does having the image tag help, or do you have a strong need for the `latest` / custom tag?

**Comment by thedrow** [comment]
I have a strong need for a latest tag. Other custom tags may be useful for other users.

**Comment by radu-matei** [comment]
Implemented in #674
